bundle/src/main/java/com/adobe/acs/commons/httpcache/config/HttpCacheConfig.java

```diff
@@ -37,7 +39,7 @@
 @ProviderType
 public interface HttpCacheConfig {
 
-    public enum FilterScope {
```

Comment:
Why did this drop in scope? This is a public package, and this go dropped to default scope; if someone were to use this enum, wouldnt this break the code?

Reply:
This is redundant.. since it's interface it's marked public by default since JDK 7.